### PR TITLE
Anishhello

### PR DESCRIPTION
**Pull request by Anish1207**
This Pull Request resolves Issue #11  

Said "Hello!" to the world in python

### CODE REVIEW

File: Binary Search/bs.txt.txt [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include<iostream.h>
```
Comment (salman-bhai) [inline Binary Search/bs.txt.txt]: Please change file extension to `bs.c`

File: Bubble Sort/Bubble Sort.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
```
Comment (salman-bhai) [inline Bubble Sort/Bubble Sort.txt]: Please change file name to `bubble.c`

File: Python/hello world.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+printf("hello world!");
```
Comment (salman-bhai) [inline Python/hello world.txt]: Change file name to `hello-world.py`
Change this line to `print("Hello World!")`

File: Participants/Anish1207.txt.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+1.name : anish patil
```
Comment (salman-bhai) [inline Participants/Anish1207.txt.txt]: Change filename to `Anish1207.txt` and add more details about yourself!

**Comment by salman-bhai** [comment]
Any updates here? Or should we close the Pull Request?

**Comment by Anish1207** [comment]
Hello

I apologise for the delay. I shall make the required changes and send a
pull request before 0000hrs tonight.


On Oct 10, 2017 1:58 PM, "Salman Shah" <notifications@github.com> wrote:

Any updates here? Or should we close the Pull Request?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/WebClub-NITK/Hacktoberfest-2k17/pull/234#issuecomment-335399719>,
or mute the thread

**Comment by salman-bhai** [comment]
Alright not an Issue!

**Pull request merged**